xbmc/pvr/timers/PVRTimerInfoTag.cpp

```diff
@@ -43,7 +43,7 @@ CPVRTimerInfoTag::CPVRTimerInfoTag(bool bRadio /* = false */) :
   m_bFullTextEpgSearch(false)
 {
   m_iClientId           = g_PVRClients->GetFirstConnectedClientID();
-  m_iClientIndex        = -1;
+  m_iClientIndex        = static_cast<unsigned int>(-1);
```

Comment:
The PR #7439 changed the type unsigned to signed. Now i get compilation warnings. Here i resolve the issue.